tests/10-migration-backend-tests/testcafe/migrations/media-islandora_object.csv

```diff
@@ -1,3 +1,3 @@
-unique_id,node_id,title,access_terms,member_of
-media_io_01,,Media Migration Repository Item One,:::media_accesscontrol_01||:::media_accesscontrol_02,:::media-collection-01
-media_io_02,,Media Migration Repository Item Two,:::media_accesscontrol_01||:::media_accesscontrol_02,:::media-collection-01
+unique_id,node_id,title,abstract,access_rights,access_terms,alternative_title,collection_number,contributor,copyright,copyright_holder,creator,custodial_history,date_available,date_copyrighted,date_created,date_published,description,digital_identifier,digital_publisher,display_hints,dspace_identifier,dspace_itemid,extent,featured_item,finding_aid,genre,geoportal_link,issn,is_part_of,item_barcode,jhir_uri,language,library_catalog_link,member_of,model,oclc_number,publisher,publisher_country,resource_type,spatial_coverage,subject,table_of_contents,title_language,years
```

Comment:
Validation: Hard to tell, but it does seem that some fields were added that are required.

The column headings are fleshed out but many of those columns don't carry any value, but it seems clear that there were a few fields in `media-islandora_object.csv` that are required but weren't present originally.

Reply:
Yes, the extra columns arise from fixing the object in the UI, then exporting.  So they're noise I didn't bother to clean up.